recipes/tuplet/all/conanfile.py

```diff
@@ -38,20 +37,22 @@ def package_id(self):
     def validate(self):
         if self.settings.compiler.get_safe("cppstd"):
             check_min_cppstd(self, self._min_cppstd)
+        check_min_vs(self, 193)
 
-        def loose_lt_semver(v1, v2):
-            lv1 = [int(v) for v in v1.split(".")]
-            lv2 = [int(v) for v in v2.split(".")]
-            min_length = min(len(lv1), len(lv2))
-            return lv1[:min_length] < lv2[:min_length]
+        if not is_msvc(self):
+            def loose_lt_semver(v1, v2):
+                lv1 = [int(v) for v in v1.split(".")]
+                lv2 = [int(v) for v in v2.split(".")]
+                min_length = min(len(lv1), len(lv2))
+                return lv1[:min_length] < lv2[:min_length]
 
-        compiler = str(self.settings.compiler)
-        version = str(self.settings.compiler.version)
+            compiler = str(self.settings.compiler)
+            version = str(self.settings.compiler.version)
 
-        minimum_version = self._compilers_minimum_version.get(compiler, False)
-        if minimum_version and loose_lt_semver(version, minimum_version):
-            raise ConanInvalidConfiguration(
-                f"{self.name} {self.version} requires C++20, which your compiler ({compiler}-{version}) does not support")
+            minimum_version = self._compilers_minimum_version.get(compiler, False)
+            if minimum_version and loose_lt_semver(version, minimum_version):
+                raise ConanInvalidConfiguration(
+                    f"{self.name} {self.version} requires C++20, which your compiler ({compiler}-{version}) does not support")
```

Comment:
same comment than https://github.com/conan-io/conan-center-index/pull/14640#discussion_r1046366965, it's convoluted.

Reply:
reverted